### ISSUE TITLE
Send email to department staff when ticket is created

### ISSUE THREAD

**Issue by keyursakaria** [opensupports/opensupports#473]
As the title says can we send email to all the staff in the particular department when ticket is created ?

**Comment by riaanvanvuuren** [comment]
I agree need to send tickets to the department staff once a ticket is created.

**Comment by mredigonda** [comment]
I agree this should be implemented, we were planning to add a notification system but even in that case, this should be implemented.

When dealing with this issue we should consider #350 too, that is somawhat related.
